e2e/claim/claim-with-request.testSuite.ts

```diff
@@ -315,29 +318,8 @@ export const claimWithRequestTestSuite = () => {
     expect(body.redirectUri).toBe(requestOrigin);
   });
 
-  it(`should throw an error when request origin is invalid`, async () => {
```

Comment:
removed this bc invalid origin will not be able reach RoleClaim constructor now since origin is checked during authentication